changes/2020-06-09_wrapping-key-identifiers/change.md

```diff
@@ -0,0 +1,147 @@
+| --------------------------------------------------------------------------------------------------- |
+| [Encrypted Data Key](../../framework/structures.md#encrypted-data-key)                              |
+| [AWS KMS Keyring](../../framework/kms-keyring.md)                                                   |
+| [Raw AES Keyring](../../framework/raw-aes-keyring.md)                                               |
+| [Raw RSA Keyring](../../framework/raw-rsa-keyring.md)                                               |
+| [Keyring Decryption Contract](https://github.com/awslabs/aws-encryption-sdk-specification/pull/131) |
+
+## Affected Specifications
+
+This serves as a reference of all specification documents that this change affects.
+
+| Specification                                             |
+| --------------------------------------------------------- |
+| [Structures](../../framework/structures.md)               |
+| [Keyring Interface](../../framework/keyring-interface.md) |
+| [AWS KMS Keyring](../../framework/kms-keyring.md)         |
+| [Raw AES Keyring](../../framework/raw-aes-keyring.md)     |
+| [Raw RSA Keyring](../../framework/raw-rsa-keyring.md)     |
+
+## Affected Implementations
+
+The scope of this change only affects the specification.
+Follow-up changes that define
+whether implementations expose these concepts
+MAY affect implementations.
+
+## Definitions
+
+### Conventions used in this document
+
+The key words
+"MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL"
+in this document are to be interpreted as described in
+[RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Summary
+
+We have referenced several terms throughout various specification documents
+that are never defined:
+"key namespace",
+"key name",
+"provider ID",
+and "provider info".
+This change defines each of these terms
+and describes their relationships to each other
+and the keyrings they represent.
+
+## Out of Scope
+
+- Whether, or how, each keyring exposes any of these values at runtime is out of scope.
+
+- Specific values for these terms for any specific keyring is out of scope.
+
+## Motivation
+
+Various specification documents reference
+"key namespace",
+"key name",
+"provider ID",
+and "provider info",
+but we never define what these terms mean
+or how they relate to each other.
+
+In order to ensure the specification documents accurately describe keyring behavior,
+we need to define all terms that they use.
+
+## Drawbacks
+
+This change SHOULD NOT introduce any drawbacks.
+It is identifying and describing
+terms that already exist
+and the existing relationship between them.
+
+## Security Implications
+
+This change SHOULD NOT have any security implications.
+
+## Operational Implications
+
+If the keyring chooses to write a provider info value that does not equal its key name,
```

Comment:
nit: this doesn't appear to be an operational implication for defining the terms "key name" and "provider info", this is just operational implications of how these concepts as they currently exist (which we aren't changing).